js/templates/modals/listingDetail/listing.html

```diff
@@ -1,161 +1,159 @@
 <div class="popInMessageHolder js-popInMessages"></div>
 
-<div class="modalContentInner">
```

Comment:
Oops, I actually made minimal changes to the file. It looks like a lot because I removed the wrapping .modalContentInner element which didn't seem necessary and had a global style in _modals.scss.